sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/VisualStudioTokenProviderFile.cs

```diff
@@ -20,9 +20,9 @@ public static VisualStudioTokenProviderFile Parse(string fileContents)
             {
                 var visualStudioTokenProviderFile = JsonHelper.Deserialize<VisualStudioTokenProviderFile>(Encoding.UTF8.GetBytes(fileContents));
 
-                // Order the providers, so that the latest one is tried first.
+                // Order the providers, so that the latest one is tried first (lowest to highest values for preference)
                 visualStudioTokenProviderFile.TokenProviders =
-                    visualStudioTokenProviderFile.TokenProviders.OrderByDescending(p => p.Preference).ToList();
+                    visualStudioTokenProviderFile.TokenProviders.OrderBy(p => p.Preference).ToList();
```

Comment:
Is there a scenario where this would be a breaking change?

Reply:
This affects which Visual Studio installation is used to get an access token when doing local developer authentication. Previously, it would choose your oldest VS installation's settings to determine which account to get a user token for. This understandably caused customer confusion/inconvenience.

As this affects only local developer scenarios and no production scenarios that this library would be used in, there's no concern here about a breaking change.